doc/mgr/dashboard.rst

```diff
@@ -1070,3 +1070,178 @@ and loosely coupled fashion.
 
 .. include:: dashboard_plugins/feature_toggles.inc.rst
 .. include:: dashboard_plugins/debug.inc.rst
+
+
+Troubleshooting the Dashboard
+-----------------------------
+
+Locating the Dashboard
+^^^^^^^^^^^^^^^^^^^^^^
+
+If you are unsure of the location of the Ceph Dashboard, run the following command::
+
+    $ ceph mgr services | grep dashboard
+    "dashboard": "https://host:port"
+
+The command returns the URL where the Ceph Dashboard is located: ``https://<host>:<port>/``
+
+
+Accessing the Dashboard
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are unable to access the Ceph Dashboard, run through the following
+commands:
+
+#. Verify the Ceph Dashboard module is enabled::
+
+    $ ceph mgr module ls
+
+#. Ensure the Ceph Dashboard module is listed in the ``enabled_modules``
+   section. Example snipped output from the ``ceph mgr module ls`` command::
```

Comment:
This is a great example to use `jq` again:
```suggestion
   section. Example snipped output from the ``ceph mgr module ls | jq .enabled_modules`` command::
```
It would return:
```
[
  "dashboard",
  "iostat",
  "restful"
]
```